scripts/generateNTPSponsoredImages.js

```diff
@@ -97,11 +97,24 @@ function downloadForRegion (jsonFileUrl, targetResourceDir) {
   })
 }
 
-async function generateNTPSponsoredImages (dataUrl) {
+function getTargetRegionList(targetRegions, excludedTargetRegions) {
+  targetRegionList = []
+  if (targetRegions === '')
```

Comment:
Maybe this should do sanitizing on the input? like stripping the spaces out

Reply:
Done. Only uppercase chars and comma are allowed.